FreeRTOS/FreeRTOS-Kernel
# Arraylist task queue

Arraylist task queue

Description
-----------
Gives an array-back implementation of lists. The new implementation does not modify the list interface. 

Test Steps
-----------
Runs with CBMC proofs. Have not tried to compile the kernel.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
